src/common/darktable.c

```diff
@@ -1799,6 +1796,10 @@ int dt_init(int argc, char *argv[], const gboolean init_gui, const gboolean load
     return 1;
   }
 
+  // set up memory.darktable_iop_names table
+  // must happen after dt_iop_load_modules_so() above
+  dt_iop_set_darktable_iop_table();
```

Comment:
Why not just move this inside `dt_iop_load_modules_so` then (and make it a local function to `imageop.c`)?